catalog/resourcemanagement/resourcemanagement-usage-plugin/pom.xml

```diff
@@ -86,22 +86,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.90</minimum>
+                                            <minimum>1.00</minimum>
```

Comment:
We should always give at least 5% wiggle room or so to account for small changes.